handler/principal_mock.go

```diff
@@ -0,0 +1,71 @@
+package handler
+
+import (
+	"github.com/yahoojapan/athenz-authorizer/v4/access"
+	"github.com/yahoojapan/athenz-authorizer/v4/role"
+)
+
+// RoleTokenMock is a mock of Principal
+type RoleTokenMock struct {
+	t role.Token
```

Comment:
```suggestion
	t role.Token
```
Can we just implement the interface? not depending the struct in `athenz-authorizer`.
Same for `OAuth2AccessTokenMock`

Reply:
Fixed
https://github.com/yahoojapan/authorization-proxy/pull/66/commits/2c72919512b30c5cdb8ef6709af43648d6fe52fe